server/v2/cometbft/config.go

```diff
@@ -27,6 +28,6 @@ type Config struct {
 
 	GrpcConfig grpc.Config
 
-	MempoolConfig
-	// CmtConfig *cmtcfg.Config
+	// MempoolConfig
+	CmtConfig *cmtcfg.Config
```

Comment:
was this intended?

Reply:
we havent completed the config stuff in comet mempool and a few other places so its probably fine to leave this as is